### ISSUE TITLE
Lost all my contact comments on Android

### ISSUE THREAD

**Issue by SkypLabs** [tutao/tutanota#7818]
- [x] This is not a feature request (existing functionality does not work, **not** missing functionality).
 I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [x] I've searched and did not find a similar issue.

**Bug in mobile app**

**Describe the bug**
I just realised that all the comments I have added to my contacts over the years, including family-related important information, are gone.

**To Reproduce**
N/A

**Expected behaviour**
Obviously, not losing your customers' data.

**Screenshots**
N/A

**Smartphone (please complete the following information):**
 - Device: Pixel 8
 - OS: GrapheneOS (Android 15)
 - Version 246.241004.0

**Additional context**
N/A

**Comment by charlag** [comment]
Hi, I'm really sorry to hear that. Did you have contact sync enabled? Did you change any settings before this happened?

**Comment by SkypLabs** [comment]
> Did you have contact sync enabled?

Yes. I have two accounts, a personal one and a professional one. I only enabled contact sync on the personal one, and I "only" lost my contact comments on this account. My contact comments on my professional account are still there.

> Did you change any settings before this happened?

No, not that I'm aware of.
